benchmark/src/main/java/org/jdbi/v3/core/qualifier/QualifiersBenchmark.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 2014, Oracle America, Inc.
```

Comment:
what's with this license header? Why is it for oracle?

Reply:
the `jmh` project generator left it, whoops.